js/src/MarketMapModel.js

```diff
@@ -69,6 +72,7 @@ var MarketMapModel = basemodel.BaseModel.extend({}, {
         axes: { deserialize: widgets.unpack_models },
         tooltip_widget: { deserialize: widgets.unpack_models },
         style: { deserialize: widgets.unpack_models },
+        layout:  { deserialize: widgets.unpack_models }
```

Comment:
note: once we get rid of basemodel, this will come from the DOMWidget serializers.